lib/mix/tasks/freebsd.pkg.ex

```diff
@@ -5,14 +5,16 @@ defmodule Mix.Tasks.Freebsd.Pkg do
 
   @shortdoc "Generates FreeBSD pkg files"
   def run(_) do
+    config = Mix.Project.config() |> Keyword.get(:freebsd, [])
```

Comment:
When is this keyword going to be missing and default to `[]`?

Reply:
Yeah, makes sense to require it here, since the library will raise in `lib/freebsd.ex` while generating the manifest.

Maybe when this config is missing, the mix task can print an error message and return a non-zero exit status, rather than raising.